tmt/utils.py

```diff
@@ -2451,6 +2451,34 @@ def get_distgit_handler_names() -> List[str]:
     return [i.usage_name for i in DistGitHandler.__subclasses__()]
 
 
+def git_clone(
+        url: str,
+        destination: str,
+        common: Common,
```

Comment:
Hm, if we made this an instance method of `Common`, we could avoid passing it here, and then we could simply call `self.parent.git_clone(...)`. What do you think? (just a nitpick and a personal taste thing, feel free to ignore)

Reply:
Well, IMHO Common is too bloated and I hope that at some point we move run elsewhere while keeping logging usable :)
If you don't mind I'll keep as is

Reply:
Yup, agreed, I also don't like `Common` very much, too much stuff there